packages/superset-ui-core/src/chart/models/ChartProps.ts

```diff
@@ -47,9 +47,6 @@ export interface ChartPropsConfig {
   height?: number;
   /** Programmatic overrides such as event handlers, renderers */
   hooks?: Hooks;
-  /** Formerly called "payload". This property going to be deprecated because
-   * contains only first item in response data array (use `queriesData` instead) */
-  queryData?: QueryData;
```

Comment:
let's remove the comment below or say something like "the data returned for all `queries` objects in the request" or similar (feel free to reword).

Reply:
updated